Troubleshooting guide for BYOK RHC alerts

karunasagark opened the pull request.

csunilkumar commented:
- As discussed you can use the template: https://github.com/hdinsight/hdinsight.github.io/edit/master/solution-template.md.
- Could you also expand the abbreviation 
- Issue is ambiguous,  talks about "disk encryption " and then about Alert. is the alert about disk encryption?
- Description starts with "This alert ensures" can we paste the alert under issue or description.
- We should be avoid words like customer.
- I would suggest you go ahead to publish this with the above changes. I will tweak the sentences to remove the "customer" in sentences.

karunasagark commented on kafka/kafka-byok-rhc.md:
```diff
@@ -0,0 +1,42 @@
+## How to fix disk encryption alerts on Resource Health Center (RHC)?
```
test comment

karunasagark commented:
test conv

karunasagark commented:
> * As discussed you can use the template: https://github.com/hdinsight/hdinsight.github.io/edit/master/solution-template.md.
> * Could you also expand the abbreviation
> * Issue is ambiguous,  talks about "disk encryption " and then about Alert. is the alert about disk encryption?
> * Description starts with "This alert ensures" can we paste the alert under issue or description.
> * We should be avoid words like customer.
> * I would suggest you go ahead to publish this with the above changes. I will tweak the sentences to remove the "customer" in sentences.

I've addressed the comments, please have a look. If you could add comments inline in the file it would help with context.
